roles/slave_cache/tasks/nodejs.yml

```diff
@@ -43,3 +43,12 @@
 - get_url: url=https://nodejs.org/dist/v10.13.0/node-v10.13.0-linux-x64.tar.xz
            dest=cache/node-v10.13.0-linux-x64.tar.xz
   when: nodejs1013.url is not defined
+
+- name: NodeJS 12.13.0 LTS
+  s3: bucket={{s3_bucket}} region={{s3_region}} object=/{{s3_tools_path}}/node-v12.13.0-linux-x64.tar.xz mode=geturl
```

Comment:
No `node-v12.13.0-linux-x64.tar.xz` archive have been put in the S3 : https://s3.console.aws.amazon.com/s3/buckets/nuxeo-devtools-resources/packages/linux/?region=eu-west-1&tab=overview

Reply:
@alexistimic How can we get them there? The source is: https://nodejs.org/dist/v12.13.0/node-v12.13.0-linux-x64.tar.xz

Reply:
I will handle it, quicker this way :)

Reply:
Thank you.